src/main/java/pignlproc/markup/AnnotatingMarkupParser.java

```diff
@@ -385,6 +387,13 @@ public void nodesToText(List<? extends Object> nodes, Appendable buffer,
                                     tagName));
                             countingBuffer.append("\n\n");
                         }
+                        if ("b".equals(tagName)) {
+                            if (tagBegin < 500) {
```

Comment:
Hi @tgalery,
Actually I want to extract bold forms only from the definition/introduction part of page, not from the whole page. If I include bold forms from whole page then it may introduce some false surface forms. Following are some examples:
1) http://en.wikipedia.org/wiki/Analysis_of_variance: "most used" and "most useful"
2) http://en.wikipedia.org/wiki/Phrase_(music): "antecedent phrase", "consequent phrase", "phrase-group" and "Phrase rhythm"
3) http://en.wikipedia.org/wiki/Afroasiatic_languages: Several bold forms in table
We can also get some valid surface forms if we extract all bold forms from page but we are not sure about validity in all cases as we have seen above.
1) http://en.wikipedia.org/wiki/Radio_Warwick: "URW312", "University Radio Warwick"